src/platform/site-wide/mega-menu/containers/Main.jsx

```diff
@@ -6,6 +6,7 @@ import { connect } from 'react-redux';
 
 // Relative imports.
 import { isLandingPageEnabled } from 'applications/mhv/landing-page/selectors';
+import { selectShowMyVALink } from 'applications/personalization/dashboard/selectors';
```

Comment:
We should move this selector out of the personalization folder and somewhere local to the site-wide/mega-menu.

The reason for doing this is that we are actually working on getting our application 'isolated' so that we can run just our CI/CD tests, and that requires that we don't import from any other apps and that no other apps/platform import from us.